R/teal_data.R

```diff
@@ -7,12 +7,12 @@
 #'
 #' @param ... any number of objects (presumably data objects) provided as `name = value` pairs.
 #'
-#' @param join_keys (`join_keys`) object or a single (`join_key_set`) object.
+#' @param join_keys (`join_keys`) optional, object or a single (`join_key_set`) object.
 #'
-#'  (optional) object with dataset column relationships used for joining.
+#'  object with dataset column relationships used for joining.
```

Comment:
The same as here https://github.com/insightsengineering/teal.data/pull/302/files#r1524668589